src/platform/datapath_raw_xdp_win.c

```diff
@@ -1088,11 +1083,18 @@ CxPlatDpRawInitialize(
     }
 
     if (CxPlatListIsEmpty(&Xdp->Interfaces)) {
-        QuicTraceEvent(
-            LibraryError,
-            "[ lib] ERROR, %s.",
-            "no XDP capable interface");
-        Status = QUIC_STATUS_NOT_FOUND;
+        if (Status == QUIC_STATUS_NOT_SUPPORTED) {
```

Comment:
This isn't the same as `HRESULT_FROM_WIN32(ERROR_FILE_NOT_FOUND)`. We could define a `QUIC_STATUS_FILE_NOT_FOUND` and use that.